.github/workflows/docker-allrpc.yaml

```diff
@@ -0,0 +1,34 @@
+name: allrpc docker
+on:
+  push:
+    branches:
+    - master
+  release:
+    types: [published]
+  pull_request:
+    branches:
+    - master
+jobs:
+  build:
+    name: build and push
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - uses: actions/checkout@v2
+    
+      - name: build
+        run: docker build --tag pionwebrtc/ion-sfu:latest-allrpc -f cmd/signal/allrpc/Dockerfile .
+
+      - name: login
+        if: github.event_name == 'release'
+        run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
+
+      - name: tag
+        if: github.event_name == 'release'
+        run: docker tag pionwebrtc/ion-sfu:latest-allrpc pionwebrtc/ion-sfu:"$TAG"-allrpc
+        env:
+          TAG: ${{ github.event.release.tag_name }}
+
+      - name: push
+        if: github.event_name == 'release'
+        run: docker push pionwebrtc/ion-sfu
```

Comment:
there were some changes to docker it seems. i believe we need to explicitly push image tags now

https://github.com/pion/ion-sfu/blob/master/.github/workflows/docker-grpc.yaml#L34

Reply:
Ok did not know about that. It should be fixed now!